### PR TITLE
Change Patreon screen

### PR DESCRIPTION
**Pull request by bogotesr**
Settings page now shows as "Patreon" rather than "Join the beta" under that has a different description. The page itself now links to the JITStreamer Patreon.

**typos have been corrected since the screenshot was taken.**

IMG_FA2006DDE46C-1

IMG_FA2006DDE46C-2

### CODE REVIEW

**Comment by SternXD** [comment]
I don't see a problem with this PR other than the typos which you said have been resolved so I see no reason not to merge this. Thoughts anyone?

**Comment by Spidy123222** [comment]
> Settings page now shows as "Patreon" rather than "Join the beta" under that has a different description. The page itself now links to the JITStreamer Patreon.
> 
> **typos have been corrected since the screenshot was taken.**
> 
> IMG_FA2006DDE46C-1
> 
> IMG_FA2006DDE46C-2

Could you make it only say SideTeam as it’s a group project, not a solo one. Other than that I’m fine with it excluding typos mentioned that were fixed.

File: AltStore/Resources/Assets.xcassets/Image.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "180 1.png",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "filename" : "180 2.png",
```
Comment (JJTech0130) [inline AltStore/Resources/Assets.xcassets/Image.imageset/Contents.json:14]: What’s up with the naming of these files? If they’re different sizes, why not name them like SideStore-1x.png? And why is the imageset just called “Image”?

File: AltStore/Resources/Assets.xcassets/jackson.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "idiom" : "universal",
+      "scale" : "1x"
+    },
+    {
+      "filename" : "unnamed.png",
+      "idiom" : "universal",
+      "scale" : "2x"
+    },
+    {
+      "filename" : "unnamed 2.png",
```
Comment (JJTech0130) [inline AltStore/Resources/Assets.xcassets/jackson.imageset/Contents.json:14]: Same thing as above, would be good to have more descriptive names

File: AltStore/Settings/PatreonViewController.swift [hunk 3]
```diff
@@ -44,9 +44,9 @@ class PatreonViewController: UICollectionViewController
         
         self.collectionView.register(aboutHeaderNib, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: "AboutHeader")
         self.collectionView.register(PatronsHeaderView.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: "PatronsHeader")
-        self.collectionView.register(PatronsFooterView.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionFooter, withReuseIdentifier: "PatronsFooter")
+        //self.collectionView.register(PatronsFooterView.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionFooter, withReuseIdentifier: "PatronsFooter")
```
Comment (JJTech0130) [inline AltStore/Settings/PatreonViewController.swift:47]: We have Git, no need to comment things out

**Review by JJTech0130 — CHANGES_REQUESTED** [review]
Other than the few things I commented on, it looks good.

**Comment by Spidy123222** [comment]
This pr works with upstream changes. I'm unsure if we should make the patreon page more broad as singling out one user can have unfair for everyone else that works hard on the project. Would it be fine to just mention SideTeam above only on left side.

**Comment by jkcoxson** [comment]
Let's get this one merged. I would prefer if y'all didn't use my picture (it's a team project) but then I think this is fine.

**Comment by Spidy123222** [comment]
i just fixed merge conflicts will test if anything needs changes

**Comment by jkcoxson** [comment]
Still not a fan of having my name and picture there

**Comment by jkcoxson** [comment]
Something is wrong with this PR, it's a duplicate of one that's already been merged. What happened to the Patreon screen changes?

**Comment by Spidy123222** [comment]
> Something is wrong with this PR, it's a duplicate of one that's already been merged. What happened to the Patreon screen changes?

Yeah this was discussed in the discord chat apparently after a accidental git revert it managed to get on their develop branch and revert to a merged pr some how.

**Comment by bogotesr** [comment]
should now be all fixed up

**Comment by JoeMatt** [comment]
32 commits?

I think this needs a rebase.

I can do it if you want in a few mins.

**Comment by JoeMatt** [comment]
I can't edit the source of a PR or publish to an outside repo, so I rebased it here.

OP can either update their branch with this or we can replace with this PR.


FYI, commit credits still belong to @bogotesr but there were conflicts with develop that needed to be resolved.

The biggest one is Server.swift is deleted instead of modified as it was removed on dev.

**Pull request closed without merging**